lms/djangoapps/instructor/views/api.py

```diff
@@ -1983,8 +1981,7 @@ def reset_student_attempts(request, course_id):
             enrollment.reset_student_attempts(course_id, student, module_state_key, delete_module=delete_module)
         except StudentModule.DoesNotExist:
             return HttpResponseBadRequest(_("Module does not exist."))
-        except sub_api.SubmissionError:
-            # Trust the submissions API to log the error
+        except: # Xblocks can throw a variety of errors, trust them to provide more detailed logging
```

Comment:
I'm wary of having a general catch here, but I figured it was better than enumerating every possible xblock error.